transmart-api-server/grails-app/conf/application.yml

```diff
@@ -1,3 +1,6 @@
+---
+patientCountThreshold: 5
```

Comment:
Please use the `org.transmartproject` prefix, just as in [transmart-notifications](https://github.com/thehyve/transmart-core/tree/dev/transmart-notifications#other-subscription-settings).